### PR TITLE
add sendmailArgs to code

### PR DESCRIPTION
**Pull request by tadr0**
Nothing for Docs yet

### CODE REVIEW

File: conf/config.pl [hunk 1]
```diff
@@ -1912,6 +1912,10 @@
 #
 $Conf{SendmailPath} = '';
 
+# Extra arguments to sendmail (as array for exec not string) for example
+# $Conf{SendmailArgs} = [qw|-C /home/backuppc/.msmtprc|] 
+$Conf{SendmailArgs} = [];
```
Comment (craigbarratt) [inline conf/config.pl:1916]: It would be better to spell this out, rather than use `perl` quotes, eg:
```
["-C", "/home/backuppc/.msmtprc"]
```

**Comment by craigbarratt** [comment]
If you look at the error messages you'll see two errors:
```
lib/BackupPC/Config/Meta.pm has SendailArgs in CgiUserConfigEdit, but conf/config.pl CgiUserConfigEdit does not
```
That looks like a typo - do you mean `SendmailArgs`?

Second, there are some `perltidy` errors, which are just formatting.  The diffs in the logs show the required changes.  It's easier if you run perltidy locally to fix it, since the whitespace needs to be exactly correct, or you do it manually.

**Comment by craigbarratt** [comment]
Ok, perltidy now passes, but the other error I mentioned remains (`SendailArgs` is misspelled).

**Comment by tadr0** [comment]
I have yet to get pre-commit running locally but a few iterations of push got it cleared up.
Thanks for  craigbarratt's patience and attention on Meta.pm.

**Comment by craigbarratt** [comment]
Can you fix the remaining test error?  I mentioned it above, and you can see it in the logs.  Thanks.

**Comment by craigbarratt** [comment]
Congrats!  You got it.  Thanks for the PR.

**Pull request merged**